jenkins/jobs/cd/dev_build.groovy

```diff
@@ -0,0 +1,71 @@
+pipelineJob('devbuild') {
```

Comment:
Just name the file as `devbuild.groovy`: keep same with name.

Reply:
done